### PR TITLE
UI, upgraded to Webpack 5

### PR DESCRIPTION
**Pull request by paoloantinori**
It builds locally but I haven't run it. If you tell me a quick way to spin the UI up I will perform further local tests.

### CODE REVIEW

File: ui/packages/registry/webpack.common.js [hunk 1]
```diff
@@ -1,18 +1,25 @@
...
+      // fixes issue with babel dependencies not declaring the package correctly for webpack 5
+      {
+        test: /\.m?js/,
+        resolve: {
+          fullySpecified: false
+        }
+      },
+      // fixes issue with yaml dependency not declaring the package correctly for webpack 5
+      {
+        test: /node_modules\/yaml\/browser\/dist\/.*/,
```
Comment (paoloantinori) [inline ui/packages/registry/webpack.common.js:20]: @EricWittmann here

**Review by carlesarnal — APPROVED** [review]
LGTM, there are test failures but I don't think they are related to this PR. The quickest way to activate the UI I'm aware of is the usual `mvn quarkus:dev` command. It will start both the quarkus app (using the in-memory variant) and the UI.

**Comment by paoloantinori** [comment]
I've seen test failures related to images too, on my own PR. Is this an infra problem? or is gh possibly subject to Docker Hub quotas?

**Comment by paoloantinori** [comment]
Ah, sorry, to give an updated. We have already validated this last night with @EricWittmann.

The process has been:
- run the companion `./init-dev.sh none` script
- manually modify `code packages/registry/src/config.js` modifying the Rest API endpoint URL to use Operate First url
- start the UI locally `yarn start:registry`

And the specific test that we have performed has been to hit the documentation page for the API (last screenshot) since that leverages the `yaml` libraries in the UI that have been problematic in the past.

Let's leave it to @EricWittmann  to decide if he wants to merge, but at the moment it seems this looks good.

Screenshot from 2021-05-18 22-27-37


Screenshot from 2021-05-18 22-29-39


Screenshot from 2021-05-18 22-32-16

**Comment by paoloantinori** [comment]
Additionally:
- I've upgraded Redoc but that's probably not required, I've done it while trying to fix another issue, but the upgrade hadn't fixed it, while the solution was to explicitly add this dependency `"to-arraybuffer": "^1.0.1",`

**Comment by EricWittmann** [comment]
This is really good.  I'll do a few minutes of extra manual testing and then merge.

**Review by EricWittmann — APPROVED** [review]

**Pull request merged**